application/models/Grocery_crud_model.php

```diff
@@ -50,7 +50,7 @@ function get_list()
     	if($this->table_name === null)
     		return false;
 
-    	$select = "`{$this->table_name}`.*";
+    	$select = "{$this->table_name}.*";
```

Comment:
also must check my comment https://github.com/scoumbourdis/grocery-crud/pull/480#issuecomment-910454201 for join with where